### DISCUSSION TITLE
Upgrading all dependencies

### DISCUSSION THREAD

**Discussion by sswatson** [astral-sh/rye#518] [Q&A]
Maybe I'm overlooking something here, but is there a way to upgrade all of a project's dependencies to the latest versions available?

The workaround I'm currently using is to remove each package and add it back. This works, but it doesn't feel like the right solution.

**Comment by rmarquis** [comment]
Have you tried `rye sync --update-all`?

**Reply by ma-sadeghi** [reply]
It'd be great if this was documented.

**Comment by sswatson** [comment]
My understanding is that that would affect the lockfile but not `pyproject.toml`? Whereas removing a package and adding it back does change the version listed in `pyproject.toml` to the newest one available.

**Reply by rmarquis** [reply]
Correct. You probably want to add a feature request here.

**Reply by AlexanderMacLennan** [reply]
This seems to still be an issue
